example/main.go

```diff
@@ -31,6 +30,7 @@ func main() {
 	item := createAndGetItem(client)
 	getAndUpdateItem(client, item.VaultID, item.ID)
 	listVaultsAndItems(client, item.VaultID)
+	validateSecretReference(item.VaultID,item.ID,"username")
```

Comment:
Let's try to format this nicer:
```suggestion
	validateSecretReference(item.VaultID, item.ID, "username")
```